api/seeders/20250128220333-seed-my-user-city.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+const MY_RECRUITER_PROFILE_ID = 455;
```

Comment:
Is this ID the same across all environments? How can we manage it to ensure it works everywhere?